processing/src/main/java/io/druid/segment/QueryableIndexStorageAdapter.java

```diff
@@ -191,7 +196,7 @@ public DateTime getMaxIngestedEventTime()
           index
       );
 
-      offset = new BitmapOffset(selector.getBitmapFactory(), filter.getBitmapIndex(selector), descending);
+      offset = new BitmapOffset(selector.getBitmapFactory(), filters[0].getBitmapIndex(selector), descending);
```

Comment:
`partitionWithBitmapSupport()` splits filters into two group, one for supporting bitmap and one for not. So we first apply bitmap filter from former and use value filter for later.